editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/provider/ImageProvider.java

```diff
@@ -209,10 +206,6 @@ public static String getKeyForLargeIcon(String iconName) {
 	}
 	
 	protected static boolean isImageAvailable(String iconName) {
-		if (CamelUtils.getDiagramEditor() != null) {
-			Image img = GraphitiUi.getImageService().getImageForId(CamelUtils.getDiagramEditor().getDiagramTypeProvider().getProviderId(), iconName);
-			return img != null;
-		}
```

Comment:
I know I put that in for a specific reason. Do u simply try to clean up code or does that part cause issues?